config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -255,16 +255,26 @@ data:
 
     # Generic OpenShift Install
     
-    - name: BootstrapFailed
-      searchRegexStrings:
-      - "Bootstrap failed to complete"
-      installFailingReason: BootstrapFailed
-      installFailingMessage: Installation Bootstrap failed to complete. Verify the networking configuration and account permissions and try again.
     - name: KubeAPIWaitTimeout
       searchRegexStrings:
       - "waiting for Kubernetes API: context deadline exceeded"
       installFailingReason: KubeAPIWaitTimeout
       installFailingMessage: Timeout waiting for the Kubernetes API to begin responding
+    - name: KubeAPIWaitFailed
+      searchRegexStrings:
+      - "Failed waiting for Kubernetes API. This error usually happens when there is a problem on the bootstrap host that prevents creating a temporary control plane"
+      installFailingReason: KubeAPIWaitFailed
+      installFailingMessage: Failed waiting for Kubernetes API. This error usually happens when there is a problem on the bootstrap host that prevents creating a temporary control plane
+    - name: BootstrapFailed
+      searchRegexStrings:
+      - "Failed to wait for bootstrapping to complete. This error usually happens when there is a problem with control plane hosts that prevents the control plane operators from creating the control plane."
+      installFailingReason: BootstrapFailed
+      installFailingMessage: Failed to wait for bootstrapping to complete. This error usually happens when there is a problem with control plane hosts that prevents the control plane operators from creating the control plane. Verify the networking configuration and account permissions and try again.
+     - name: BootstrapFailed
```

Comment:
I think you have an extra space here that's causing the UT failure.

Also, I'm pretty sure you're going to need to give this a different `name` than the previous one.